src/Sql/Sql/TransparentDataEncryption/Cmdlet/RemoveAzureRmSqlManagedInstanceKeyVaultKey.cs

```diff
@@ -0,0 +1,69 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.Common.Authentication.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Sql.Common;
+using Microsoft.Azure.Commands.Sql.Server.Model;
+using Microsoft.Azure.Commands.Sql.TransparentDataEncryption.Adapter;
+using Microsoft.Azure.Commands.Sql.TransparentDataEncryption.Model;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Security;
+
+
+namespace Microsoft.Azure.Commands.Sql.TransparentDataEncryption.Cmdlet
+{
+    /// <summary>
+    /// Defines the Remove-AzureRmSqlManagedInstanceKeyVaultKey cmdlet
+    /// </summary>
+    [Cmdlet("Remove", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlInstanceKeyVaultKey", SupportsShouldProcess = true, DefaultParameterSetName = DefaultParameterSet)]
+    [OutputType(typeof(AzureRmSqlManagedInstanceKeyVaultKeyModel))]
```

Comment:
Generally, remove cmdlets return nothing if successful.  Is this pattern followed in other Remove cmdlets in Sql?

Reply:
Yes, I have followed the pattern in the released cmdlet for sql server as we want to keep the user experience same for sql server and managed instance. The corresponding cmdlet for [sql server key vault key](https://docs.microsoft.com/en-us/powershell/module/az.sql/remove-azsqlserverkeyvaultkey?view=azps-2.2.0) returns the deleted key and [remove key vault key cmdlet](https://docs.microsoft.com/en-us/powershell/module/azurerm.keyvault/remove-azurekeyvaultkey?view=azurermps-6.13.0) also returns deleted key.